llvm/utils/update_mc_test_checks.py

```diff
@@ -118,6 +119,19 @@ def main():
         default=None,
         help="Set a default -march for when neither triple nor arch are found in a RUN line",
     )
+    parser.add_argument(
+        "--remove-duplicate",
```

Comment:
Maybe just call it `--unique` (by vague analogy with Unix commands `uniq` and `sort -u`).

Reply:
good idea. Use unique now